lib/libyui_client/process_helper.rb

```diff
@@ -3,19 +3,28 @@
 require 'timeout'
 require 'socket'
 require 'rainbow'
+require 'libyui_client/config_reader'
 
 # Client to interact with YAST UI rest api framework for integration testing
 module LibyuiClient
   # default timeout for process
   DEFAULT_TIMEOUT_PROCESS = 20
 
+  def self.set_host
+    ENV['YUI_HTTP_HOST'] ||= read_config_value('YUI_HTTP_HOST')
+  end
+
   # set the application introspection port for communication
   def self.set_port
     # 14155 is currently an unassigned port
-    ENV['YUI_HTTP_PORT'] ||= '14155'
+    ENV['YUI_HTTP_PORT'] ||= read_config_value('YUI_HTTP_PORT')
     ENV['YUI_HTTP_PORT']
   end
 
+  def self.set_api_version
```

Comment:
a another one here would not hurt :) But we can also do it later, I see that in other files I missed some comments too :) just to consistent with what is already in the file but not an important thing at the moment.